validator/src/main.rs

```diff
@@ -636,14 +643,14 @@ pub fn main() {
                 .help("Comma separated persistent accounts location"),
         )
         .arg(
-            clap::Arg::with_name("gossip_port")
+            Arg::with_name("gossip_port")
```

Comment:
Follow this newly added line: https://github.com/solana-labs/solana/pull/9161/files#diff-1f422d7e73618aca7bce9935356de012R807